hipify-clang/packaging/hipify-clang.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(hipify-clang)
 
-install(PROGRAMS @PROJECT_BINARY_DIR@/hipify-clang/hipify-clang DESTINATION bin)
+install(PROGRAMS @PROJECT_BINARY_DIR@/hipify-clang DESTINATION bin)
```

Comment:
I do not understand the necessity of install procedure call here, is install commands in hipify-clang's CMakeLists.txt not enough?